new-lamassu-admin/src/pages/Wallet/helper.js

```diff
@@ -1,16 +1,22 @@
 import * as R from 'ramda'
 import * as Yup from 'yup'
 
+import { NumberInput } from 'src/components/inputs/formik'
 import Autocomplete from 'src/components/inputs/formik/Autocomplete.js'
 
 const filterClass = type => R.filter(it => it.class === type)
 const filterCoins = ({ id }) => R.filter(it => R.contains(id)(it.cryptos))
-
+const currencyMax = 999999999
```

Comment:
Probably it's out of the scope of this card but we should consider moving this 'currency max' to a single place since it's used in many places. What you think?

Reply:
I agree, let me message Rafael